test/helpers/create.js

```diff
@@ -2,5 +2,5 @@ var hypercore = require('../../')
 var memdb = require('memdb')
 
 module.exports = function create () {
-  return hypercore(memdb())
+  return hypercore(memdb({keyEncoding: 'json'}))
```

Comment:
to make the tests prove that you can supply any key encoding. should i add a comment?